src/adapters/bidderFactory.js

```diff
@@ -235,6 +236,10 @@ export function newBidder(spec) {
         onBid: (bid) => {
           const bidRequest = bidRequestMap[bid.requestId];
           if (bidRequest) {
+            if (isUnknownBidder(bid.bidderCode, bidRequest.bidder, bid.adapterCode)) {
```

Comment:
I don't think it makes sense to look for `bid.adapterCode` - we shouldn't need to ask adapters to set that. Core can determine`adapterCode` by looking for the root of the alias tree:

1. set `adapterCode = bidRequest.bidderCode`
2. if `aliasRegistry[adapterCode]`, set `adapterCode = aliasRegistry[adapterCode]` and goto 1
3. set `bid.adapterCode = adapterCode`

I think this would work because the aliasing logic should prevent cycles.

Reply:
Using `bidRequest.bidder` makes sense. Thank you :)